### ISSUE TITLE
PRO :: `marker-listing.js`

### ISSUE THREAD

**Issue by shazahm1** [CodeCabin/wp-google-maps#1018]
In the `WPGMZA.MarkerListing.prototype.onItemClick` function the code does not check for a `null` return type when calling `map.getMarkerByID(marker_id)` which  causes an error.

The code as it is in version `8.1.20`.

``` javascript
		var marker = this.map.getMarkerByID(marker_id);
		var listingPushedInMap = WPGMZA.maps[0].settings.push_in_map && WPGMZA.maps[0].settings.push_in_map.length;
		var clickedGetDirections = $(event.target).hasClass("wpgmza_gd");
		var zoomLevelOnClick = 13;
		
		marker.trigger("select");
```

Should be:

``` javascript
		var marker = this.map.getMarkerByID(marker_id);
		var listingPushedInMap = WPGMZA.maps[0].settings.push_in_map && WPGMZA.maps[0].settings.push_in_map.length;
		var clickedGetDirections = $(event.target).hasClass("wpgmza_gd");
		var zoomLevelOnClick = 13;

		if ( null === marker ) {
			return;
		}
		
		marker.trigger("select");
```

**Comment by shazahm1** [comment]
@DylanCodeCabin

**Comment by DylanCodeCabin** [comment]
@shazahm1 This will be added to the V9 builds, alongside: 
- https://github.com/CodeCabin/wp-google-maps/issues/1017

**Comment by DylanCodeCabin** [comment]
This has been patched in our V9.0.0 builds, which have a confirmed release date of `2022-06-20`

Thank you @shazahm1
